lib/pushmi_pullyu/aip/file_list_creator.rb

```diff
@@ -20,6 +21,9 @@ class ListSourceFileSetMismatch < StandardError; end
 
   def initialize(list_source_uri, output_xml_file, file_set_uuids)
     @uri = RDF::URI(list_source_uri)
+    @auth_uri = RDF::URI(list_source_uri)
+    @auth_uri.user = PushmiPullyu.options[:fedora][:user]
```

Comment:
@uri is also used as the subject for the triples, so it can't contain the user/pass in the uri. It needs a auth_uri to just allow the graph to load.